config.py

```diff
@@ -780,7 +780,7 @@
 Whether independent component analysis should be used or not.
 """
 
-ica_reject: Optional[Dict[str, float]] = None
+ica_reject: Optional[Union[Dict[str, float], Literal['auto']]] = 'auto'
```

Comment:
I would put None by default at least for now. Autoreject can lead to quite time consuming jobs.